### DISCUSSION TITLE
How to generate a texture atlas with Graphite ?

### DISCUSSION THREAD

**Discussion by qwerasdfzxcvqs** [BrunoLevy/GraphiteThree#44] [Q&A] [answered]
Can you tell me how to use Graphite to create such a texture arrangement?
![ec10fb7265ff67f7c396f1865642de57](https://github.com/BrunoLevy/GraphiteThree/assets/101967777/936acdc5-891a-44db-a6f5-c3515e1966aa)

**Comment by BrunoLevy** [comment, marked as answer]
Sure ! An image is worth 1000 words (and an animation is worth 24000 words per second !):
![GraphiteMakeAtlas](https://github.com/BrunoLevy/GraphiteThree/assets/19478253/37a35c6c-2ef0-4e0e-93c9-3ca2ce0a5481)

**Comment by qwerasdfzxcvqs** [comment]
Ok，thank you very much.And I have another question, can the pixel size of the texture arrangement map generated by permutation be specified, or can it be set to be arranged on multiple maps？

**Comment by BrunoLevy** [comment]
- there is no option to generate multiple textures from a single object. But you can separate your object into multiple parts, and generate a texture for each part, then merge the parts together.
- all texture coordinates are between 0 and 1. There is no such a thing as 'pixel size' at this step of the process.
- for now, all textures are supposed to be square. There may be some options in the xatlas library that I'm using for specifying rectangle textures.
